src/models/logic/cccChange.ts

```diff
@@ -162,7 +162,7 @@ export async function getByAddress(
             },
             limit: itemsPerPage,
             offset: (page - 1) * itemsPerPage,
-            order: [["id", "ASC"]]
+            order: [["blockNumber", "DESC"]]
```

Comment:
Descending order looks fine, but sorting by block number can make the result unstable.
How about sorting by block number and id?

Reply:
I though descending order is fine.
However, I think keeping it ascending order would be better because you added count API.
If you keep it an ascending order, the same URL would return the same result at any time.
How about keeping it ascending order and make the explorer calculate the last page with the count?

Reply:
I think that's a good idea. But, with the current pagination parameters(page/itemsPerPage), it's impossible to get exact N items for the first page(in the explorer's point of view) if I do that.

For consistency with other APIs, I think it's better to leave it as later work.